### PR TITLE
refactor: remove metamask

### PR DESCRIPTION
**Pull request by bukandicki**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-2891

### Changelog/Description 🗒️
- remove metamask
- enhancement dynamic disable menu GA on update verification

### WARNING ⚠️
-

### CODE REVIEW

File: src/store/metamask/index.js [hunk 1]
```diff
@@ -1,49 +1,25 @@
 import Web3 from "web3"
-import contracts from "./contracts"
 import store from "@/store/index"
 
 const defaultState = {
```
Comment (agustinustheo) [inline src/store/metamask/index.js:4]: Why is this file still here?

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/common/components/Navbar.vue [hunk 2]
```diff
@@ -153,21 +126,18 @@ export default {
...
     searchQuery: "",
     contentHover: false,
     loginStatus: false,
     debounce: null,
     arrowPosition: "",
-    showMetamaskDialog: false,
     balance: 0,
     walletAddress: "",
     activeBalance: 0,
     ethAccount: null,
```
Comment (ciniiia) [inline src/common/components/Navbar.vue:139]: please remove this

File: src/common/components/Navbar.vue [hunk 3]
```diff
@@ -153,21 +126,18 @@ export default {
...
     loading: false,
-    activeMetamaskAddress: null,
     ethRegisterAddress: null,
```
Comment (ciniiia) [inline src/common/components/Navbar.vue:141]: also this one, please remove

File: src/common/components/Navbar.vue [hunk 4]
```diff
@@ -340,66 +273,11 @@ export default {
...
-
-    handleDropdownAction(type) {
-      if (type === "metamask") this.disconnectWallet()
-    },
-
     signOut () {
       localStorage.clear()
       this.clearAuth
-      this.clearWallet()
       this.ethAccount = null
```
Comment (ciniiia) [inline src/common/components/Navbar.vue:279]: please remove this unused line

**Review by ciniiia — CHANGES_REQUESTED** [review]

**Review by ciniiia — APPROVED** [review]

**Pull request merged**